packit_service/worker/reporting.py

```diff
@@ -190,6 +198,11 @@ def _add_commit_comment_with_status(
             )
             + f"\n\n{description}"
         )
+        if not self.is_final_state(state):
+            # To avoid multiple comments for non-final states
+            logger.debug(f"Not a final state -> not adding a comment with {body}")
```

Comment:
```suggestion
            logger.debug(f"Not a final state -> not adding a comment with body:\n{body}")
```

I am not sure if we want to log the whole body but it shouldn't happen so often so I am not against it.

Reply:
I wouldn't say that taking in account the current state of GitHub ;)

Reply:
I've shortened it.